clang/lib/StaticAnalyzer/Checkers/WebKit/RawPtrRefCallArgsChecker.cpp

```diff
@@ -1,4 +1,4 @@
-//=======- UncountedCallArgsChecker.cpp --------------------------*- C++ -*-==//
+//=======- RawPtrRefCallArgsChecker.cpp --------------------------*- C++ -*-==//
```

Comment:
Would it make sense to have documentation for `RawPtrCallArgsChecker` in `checkers.rst`? Do you anticipate more generic checkers calling this checker besides `UncountedCallArgsChecker` and `UncheckedCallArgsChecker`?

Reply:
I don't think so. I could imagine `UncountedCallArgsChecker` and `UncheckedCallArgsChecker` getting merged in the future but I don't think we'd add a new checker using `RawPtrCallArgsChecker`.